oauth2client/client.py

```diff
@@ -1222,6 +1227,31 @@ def create_scoped(self, scopes):
         """
         return self
 
+    @classmethod
+    def from_json(cls, s):
+        from oauth2client.service_account import _ServiceAccountCredentials
+        s = _from_bytes(s)
+        data = json.loads(s)
+
+        # A GoogleCredentials.from_stream return value might be a
```

Comment:
I don't understand this comment at all - we're implementing `from_json`, not `from_stream`. What does `from_stream` have to do with anything here?